### PR TITLE
modify sendEmail

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: naples/utils.py [hunk 1]
```diff
@@ -84,7 +87,7 @@ def createMsgEmailChangePassword(token: str, verify_router: str) -> str:
     return html_content
 
 
-def sendEmail(email: str, message: str, ses_client: SESClient):
+def sendEmail(email: str, message: str, ses_client: SESClient, db: Session):
```
Comment (Taraslysak) [inline naples/utils.py]: So if we fail to send ANY email, we delete the user????

Comment (Taraslysak) [reply naples/utils.py]: why do we need to pass user email as argument, if we can pass user model directly?

**Review by Taraslysak — COMMENTED** [review]
Please take a close look to my comments

File: naples/utils.py (hunk 1 above)
Comment (varvarashcherbyna7) [reply naples/utils.py]: yes, you are right, I will add the user model

File: naples/routes/auth.py [hunk 2]
```diff
@@ -117,7 +128,25 @@ def sign_up(
...
+            recipient_email=new_user.email,
+            sender_email=CFG.MAIL_DEFAULT_SENDER,
+            message=msg,
+            charset=CFG.CHARSET,
+            mail_body_text=CFG.MAIL_BODY_TEXT,
+            mail_subject=CFG.MAIL_SUBJECT,
+        )
+        sendEmailAmazonSES(emailContent, ses_client=ses)
+
+    except Exception as e:
```
Comment (Taraslysak) [inline naples/routes/auth.py]: Not all exceptions should be intercepted here, but only those related to email sending

**Review by Taraslysak — COMMENTED** [review]
Just a little bit more refactoring please =)

File: naples/routes/auth.py (hunk 2 above)
Comment (varvarashcherbyna7) [reply naples/routes/auth.py]: ok)

**Review by Taraslysak — APPROVED** [review]

**Pull request merged**